internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -2057,12 +2057,10 @@ func configReplicaSetScalingStrategyOldSchema(orgID, projectName, name, replicaS
 		data "mongodbatlas_advanced_cluster" "test" {
 			project_id = mongodbatlas_advanced_cluster.test.project_id
 			name 	     = mongodbatlas_advanced_cluster.test.name
-			use_replication_spec_per_shard = true
```

Comment:
are we testing this attribute in other tests?

Reply:
Yes it's being tested in other tests. In this case I removed it to make sure that `replica_set_scaling_strategy` is set even if the flag is set to false(default value)